### PR TITLE
Sup 3498 disable kernel patch

### PR DESCRIPTION
**Pull request by kaloyan-mm**
removed unnecessary opts in the setup

### CODE REVIEW

File: .github/linters/install_chrome.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+apt-get -y update
+apt-get -y upgrade
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+apt-get install ./google-chrome-stable_current_amd64.deb
```
Comment (srushe) [inline .github/linters/install_chrome.sh:6]: Looks like we need `-y` on this as well, to confirm installing extra packages.

```suggestion
apt-get -y install ./google-chrome-stable_current_amd64.deb
```

File: Gemfile [hunk 2]
```diff
@@ -69,3 +69,5 @@ if (agent_path=ENV['AGENT_PATH'])
 else
    gem 'contrast-agent' if !!ENV['CI_TEST']
 end
+
+gem 'puma' # TODO: RUBY-1541; remove puma dependency for capybara
```
Comment (srushe) [inline Gemfile:73]: Will this cause any bother with our conditional installations of puma for `puma_max` and `puma_min`?

File: Gemfile.lock [hunk 3]
```diff
@@ -180,28 +193,31 @@ GEM
     parser (3.1.0.0)
       ast (~> 2.4.1)
     path_expander (1.1.0)
-    racc (1.5.2)
+    public_suffix (4.0.6)
+    puma (5.6.2)
```
Comment (srushe) [inline Gemfile.lock:197]: I'm intrigued to see how this plays with the `gem 'puma', '~> 3.7'` requirement for `puma_min`.

File: spec/app/controllers/vulneruby_engine/static_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        options.add_argument("--headless")
+        options.add_argument("--no-sandbox")
+        options.add_argument("--disable-dev-shm-usage")
+        options.add_argument("--window-size=1280,900")
+        Capybara::Selenium::Driver.new(app, browser: :chrome, capabilities: options)
+      end
+      Capybara.javascript_driver = :headless_chromium
+    end
+
+    it 'does whatever this is' do
```
Comment (srushe) [inline spec/app/controllers/vulneruby_engine/static_controller_spec.rb:21]: 🤨

```suggestion
    it 'retrieves the static page' do
```

**Review by srushe — COMMENTED** [review]
I assume this is actually for RUBY-1545, despite the title and description, mainly because I see nothing about Kernel in here. Some questions, but assuming they're all fine I'll approve this.

File: spec/app/controllers/vulneruby_engine/static_controller_spec.rb (hunk 4 above)
Comment (kaloyan-mm) [reply spec/app/controllers/vulneruby_engine/static_controller_spec.rb:21]: I am not done with the test, more I'm trying to make it work in the pipeline :D

**Review by hmcginnis — APPROVED** [review]

**Review by Teodor-Raychev — APPROVED** [review]

**Pull request merged**